### PR TITLE
Feature ver alertas 159956428

### PR DESCRIPTION
**Pull request by FedePe**
PR story "View Alerts".
The alerts assigned to the user are displayed. These alerts can be assigned directly to the user or to a group that he belongs to.
From each alert you can see its description and the creator.
The type of the alert determines the style of the alert.

### CODE REVIEW

File: src/components/ViewAlerts.jsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+  componentDidMount() {
+    const endPoint = `${process.env.REACT_APP_API_URL}/alerts`;
+    fetch(endPoint, {
+      headers: new Headers({
+        Authorization: `Bearer ${window.localStorage.getItem('sessionToken')}`,
+      }),
+    })
+      .then((res) => {
+        if (res.status === 404) throw Error('The user does not have alerts.');
```
Comment (elcvilla) [inline src/components/ViewAlerts.jsx]: The error does not correspond to the message

Comment (elcvilla) [inline src/components/ViewAlerts.jsx]: The error does not correspond to the message

Comment (jpducassou) [reply src/components/ViewAlerts.jsx]: Totally agree!

File: src/components/ViewAlert.jsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+import React from 'react';
+
+export default function Alert(props) {
+  const {
+    type, description, nickname, last_name: lastName, first_name: firstName,
+  } = props;
+  if (type === 'info') {
+    return (
+      <div className="view-alert">
+        <div className="alert alert-info media">
```
Comment (jpducassou) [inline src/components/ViewAlert.jsx]: Couldn't you just interpolate the type variable?

File: src/components/ViewAlert.jsx [hunk 3]
```diff
@@ -0,0 +1,73 @@
+import PropTypes from 'prop-types';
+import '../css/ViewAlert.css';
+import React from 'react';
+
+export default function Alert(props) {
```
Comment (jpducassou) [inline src/components/ViewAlert.jsx]: You must name the Component the same way as the file

**Pull request merged**